src/util/isIE8.js

```diff
@@ -0,0 +1,10 @@
+var isIE8 = function () {
```

Comment:
Wrap this function in an IIFE (`(...)()`) so that it executes immediately and the value of `isIE8`, the exported value, is a boolean. We only need to execute the function once during runtime.